### ISSUE TITLE
Improve `/community` Testimonial Section

### ISSUE THREAD

**Issue by julieg18** [iterative/dvc.org#3092]

**Comment by Rutuja-Kelkar** [comment]
Hi @julieg18 

The issue regarding testimonials interested me and hence I created a improved design which is unique and user centric, Please let me know what you think of the new testimonial design.

Please take a look at the figma file : [https://www.figma.com/file/olPIVI1MgwV5PRAjDfNa0x/DCV_design_OpenSource?node-id=0%3A1](url)


### Hers's how the new testimonial section looks :

![TestimonialSectionFrame](https://user-images.githubusercontent.com/66067675/147492622-4ddb5490-a026-4bde-a696-ca1698ff88e6.png)

### Here's a logo for testimonial section : 

![TstimonialIcon](https://user-images.githubusercontent.com/66067675/147492661-19b7eed5-ab11-4933-8281-d30b759de172.png)


Cheers :)
Rutuja Kelkar
[https://twitter.com/therutujakelkar](url)
https://www.rutujakelkar.com/

**Comment by julieg18** [comment]
@Rutuja-Kelkar, I like the design! Unfortunately, we are using a [Testimonial](https://testimonial.to/) iframe which isn't editable. This ticket is about improving the design around the iframe itself. Sorry for the confusion! 

As for the icon: @iterative/websites, what do you think?

**Comment by rogermparent** [comment]
> As for the icon: @iterative/websites, what do you think?

An alternate icon could be good considering we have a duplicate between community and testimonials, but if we adopt this one we'll want to add a rounded square background to keep it consistent with the other icons.

**Comment by yathomasi** [comment]
I agree with @rogermparent  for icon. We would want to keep the consistent design with other icons. For color, it looks like we have three of them which we can alternate so that no two color are in a row.

**Comment by jendefig** [comment]
@Rutuja-Kelkar This design looks really great!  Thanks for your idea and contribution!  Our testimonials feed from  Testimonial.to.  I'm not sure how that affects the ability to adjust the design.  Ultimately we will have other types of Testimonials besides Tweets (video, text, LinkedIn), so not sure how that would work in this format too.  But this does look better and I like how they scroll horizontally to not take so long to get to the other sections of the Community page when you scroll down. Great work!

I will say I prefer the testimonial icon we have.  This one is cute, but resembles Pac-Man. 

@julieg18
